server/@types/contactsApi/index.d.ts

```diff
@@ -4325,6 +4325,39 @@ export interface components {
       property?: string
       ignoreCase?: boolean
     }
+    /** @description Describes the prisoner contact relationship */
+    PrisonerContactRelationshipDetails: {
+      /**
```

Comment:
Are we limiting to currentTerm here? I.e they relate to the current (or latest) term in prison?

Reply:
It's referenced by id so I guess you could load it but they shouldn't be returned in the contact list right?